topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md

```diff
@@ -0,0 +1,699 @@
+{: .comment}
+
+
+## Run workflow on different data
+
+Now that we have built our workflow, let's use it on some different data. For example, let's describing `diamonds` R dataset.
+
+> ### {% icon hands_on %} Hands-on: Create a new history and upload a new data
+>
+> 1. Create a **new history** and give it a name.
+>
+>    {% include snippets/create_new_history.md %}
+>
+> 2. **Import** {% icon galaxy-upload %} the file `diamond.csv` from [Zenodo](https://zenodo.org/record/3522106/files/diamonds.csv) or from the data library (ask your instructor)
+>
+>    ```
+>    https://zenodo.org/record/3522106/files/diamonds.csv
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+> 3. **Rename** {% icon galaxy-pencil %} the dataset to `diamonds`
+>
+>    {% include snippets/rename_dataset.md %}
+>
+> 4. Add a propagating tag {% icon galaxy-tags %} (e.g. `#diamonds`)
+>
+>    {% include snippets/add_tag.md %}
+{: .hands_on}
+
+The `diamonds` data set comes from the well-known ggplot2 package developed by Hadley and initially collected from Diamond Search Engine in 2008.
+The dataset contains 53940 observations and 10 variables within the 4 Cs (carat, cut, color and clarity):
+  - **Carat** refers to the weight of the diamond when measured on a scale
+  - **Cut** applies to the quality of the cut (Fair, Good, Very Good, Premium, Ideal)
+  - **Color**, is a qualitative data graded on a letter scale from D to Z (D being the best, known as colorless), color describes the overall tint, or lack thereof, of the diamond, from colorless/white to yellow
+  - **Clarity** describes the amount and location of naturally occuring "inclusions" found in nearly all diamonds, a measurement of how clear the diamond is (I1 (worst), SI2, SI1, VS2, VS1, VVS2, VVS1, IF (best))
+
+![4 Cs](../../images/4cs-of-diamond-buying.jpg "The 4 Cs of diamond of diamonds.")
+
+> ### {% icon hands_on %} Hands-on: Run workflow
+>
+> 1. We wish to analyze diamonds by their 4 Cs by recycling our workflow.
+>
+> 2. Open the **workflow menu** (top menu bar).
+>    - Find the workflow you made in the previous section,
+>    - Select the option `Run`.
+>    - The central panel will change to allow you to configure and launch the workflow.
+>
+> 3. Select the appropriate dataset for the input, change column number to the different tools and create 2 scatterplots representing price according to carat with either color or clarity as factor
+>
+> 4. Click `Run workflow`.
+>
+> 5. Once the workflow has started, you will initially be able to see all its steps, but the unimportant intermediates will disappear after they complete successfully:
+{: .hands_on}
+
+> ### {% icon question %} Questions
+> 1. How many cut category are there in the Diamond dataset ?
+> 2. How many samples are there in each cut category ?
+> 3. Which are the metrics for carat, depth, table, price, x, y and z ?
```

Comment:
@nagoue or @annefou could you add the answers to the questions in this box as well?

Reply:
@shiltemann I added answers to the questions. For the last question "metrics for carat, depth, table, price, x, y and z ", the formatting is not very nice (the table is too large) but I am not sure how to proceed to make it nicer. Do you have a suggestion?

Reply:
Hi @annefou thanks a lot for adding it! 

For the formatting: putting it in a literal block (``` before and after) could work, but another solution could be to maybe just show (one of) the scatterplot(s) instead of the table, and ask if they can observe a clear separation of classes? what do you think?

Reply:
Hi @shiltemann Thanks for your suggestion! I updated the last question (and answer) to focus on usng the scatter plot. (so no more table and formatting problem!). What do you think about it?

Reply:
looks great, thanks a lot! I will push some final tweaks today, then I think we can merge, is it ready to go from your side as well? :)

Reply:
Yes it is ready to go on our side. Thanks a lot for your very useful feedback and review.